validators/compliance-requirements.adoc

```diff
@@ -0,0 +1,110 @@
+= Validator Node Compliance Requirements
```

Comment:
General idea: We also should add telegram notification to not in compliance/compliance so they will get notified. Or give an example here where can they get this data from solidity so they will add it to their bots.